src/scripts/main.js

```diff
@@ -1,3 +1,391 @@
 'use strict';
+'use strict';
+
+const tbody = document.querySelector('tbody');
+const title = document.querySelector('thead');
+let revers = '';
+
+const handleActiveClas = () => {
+  tbody.querySelectorAll('tr').forEach((item) => {
+    item.addEventListener('click', (e) => {
+      const isActive = document.querySelector('.active');
+
+      if (isActive) {
+        isActive.classList.remove('active');
+      }
+
+      e.currentTarget.classList.add('active');
+    });
+  });
+};
```

Comment:
The function name 'handleActiveClas' may be a typo. Consider renaming it to 'handleActiveClass' for better readability.